.gitignore

```diff
@@ -14,3 +14,9 @@ __debug_bin*
 llama/build
 llama/vendor
 /ollama
+
+# Local dev
+bin/
+*.log
+*.pid
```

Comment:
Would it be possible to remove these as part of the PR (or open a separate one) to keep it focused?